sos/plugins/ceph.py

```diff
@@ -8,14 +8,16 @@
 
 from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
 from socket import gethostname
+import re
 
 
 class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
     """CEPH distributed storage
     """
 
     plugin_name = 'ceph'
-    profiles = ('storage', 'virt')
+    profiles = ('storage', 'virt', 'container')
+    containers = ('ceph-(mon|rgw|osd)*',)
```

Comment:
Sadly, legacy-3.9 does not have the ability to enable a plugin based on a container presence. Cf. https://github.com/sosreport/sos/blob/legacy-3.9/sos/plugins/__init__.py#L1603 .

https://github.com/sosreport/sos/commit/c10c1a39a338cb52ca398a5568b1dc31001612fa needs to be backported from main to legacy-3.9 to have this working.

Reply:
Oh, good catch!